### PR TITLE
Delay mass assignments

### PR DESCRIPTION
**Pull request by dschaub**
/domain @samandmoore @while1malloc0 @agirlnamedsophia

### CODE REVIEW

**Comment by while1malloc0** [comment]
<< domain

File: app/models/bulk_assignment_creation.rb [hunk 1]
```diff
@@ -25,6 +25,10 @@ def save
     true
   end
 
+  def save!
+    save || raise(ActiveRecord::RecordNotValid)
```
Comment (while1malloc0) [inline app/models/bulk_assignment_creation.rb]: Should this be `raise ActiveRecord::RecordInvalid, self`?

**Comment by while1malloc0** [comment]
<< domain lgtm

File: app/models/bulk_assignment_creation.rb (hunk 1 above)
Comment (samandmoore) [inline app/models/bulk_assignment_creation.rb]: (i think so)

**Comment by samandmoore** [comment]
<<domainlgtm

**Comment by while1malloc0** [comment]
Oh right, I'm not on test track core. Derp 😄

**Pull request merged**